docs/tutorials/control/air-quality-fleet.md

```diff
@@ -0,0 +1,809 @@
+
+As you'll find out in the [authentication step](#authenticate-your-code-to-your-viam-app-location), you can set each customer up with credentials to access the data from only their location, or you can create a dashboard showing data from all sensors in your entire organization.
+
+{{<imgproc class="aligncenter" src="/tutorials/air-quality-fleet/two-sensors.png" resize="x1000" declaredimensions=true alt="The air quality dashboard you'll build. This one has PM2.5 readings from two different sensor machines displayed, and a key with categories of air quality." style="max-width:500px" >}}
+
+### Set up your TypeScript project
+
+Complete the following steps on your laptop or desktop.
+You don't need to install or edit anything else on your machine's single-board computer (aside from `viam-server` which you already did); you'll be running the TypeScript code from your personal computer.
+
+1.  Make sure you have the latest version of [Node.JS](https://nodejs.org/en) installed on your computer.
+1.  Install the Viam TypeScript SDK by running the following command in your terminal:
+
+    ```sh {class="command-line" data-prompt="$"}
+    npm install --save @viamrobotics/sdk
+    ```
+
+1.  Create a directory on your laptop or desktop for your project.
+    Name it <file>aqi-dashboard</file>.
+
+1.  Create a file in your <file>aqi-dashboard</file> folder and name it <file>package.json</file>.
+    The <file>package.json</file> file holds necessary metadata about your project.
+    Paste the following contents into it:
+
+    ```json {class="line-numbers linkable-line-numbers"}
+    {
+      "name": "air-quality-dashboard",
+      "description": "A dashboard for visualizing data from air quality sensors.",
+      "scripts": {
+        "start": "esbuild ./main.ts --bundle --outfile=static/main.js --servedir=static --format=esm",
+        "test": "echo \"Error: no test specified\" && exit 1"
+      },
+      "author": "Viam Docs Team",
+      "license": "ISC",
+      "devDependencies": {
+        "esbuild": "*"
+      },
+      "dependencies": {
+        "@viamrobotics/sdk": "^0.13.0",
+        "bson": "^6.6.0"
+      }
+    }
+    ```
+
+{{% alert title="Fun fact" color="info" %}}
+The `--format=esm` flag in the `"start"` script is important because the ECMAScript module format is necessary to support the BSON dependency this project uses for data query formatting.
+If you don't know what the proceeding sentence means, don't worry about it; just copy-paste the JSON above and it'll work.
+{{% /alert %}}
+
+### Authenticate your code to your Viam app location
+
+Your TypeScript code requires an API key to establish a connection to your machines.
+
+You can set up credentials to access data from all the sensor machines in your organization, or from just one location, so in our example you could provide Antonia with an API key to see the data from her location, and provide RobotsRUs with a different API key to access the data from their RobotsRUs location.
```

Comment:
Reflecting a bit more. This tutorial doesn't address whether I ever grant Antonia or RobotsRUs access to their respective location in the app. I guess what you are saying here is we would just give them access to the hosted webpage? Also if they are non technical what does Antonia do with the provided API key?

Reply:
Yeah you're right--I wasn't sure exactly how to present this. Since we aren't covering provisioning (except with a vague note), this isn't exactly what someone would probably do in prod anyway; it's somewhere in between personal use and prod. They could send the code repo link to Antonia, and the API key, and have her replace the strings with the key and spin up the website herself. Or, they could put her location API key in and host the website somewhere for her but this tutorial doesn't cover hosting at all. Or maybe they could change the website to have a login screen where her login maps to a set of API keys stored on the server side where they're hosting, and they never actually give her the API key? I don't know much about hosting or auth so idk if that would make any sense.

Reply:
But basically I was hoping that's up to the reader if they're actually doing something beyond a hobby project with this.